README.md

```diff
@@ -108,7 +108,6 @@ In addition to [the parameters that can be configured for the PostgreSQL JDBC Dr
 |`failoverClusterTopologyRefreshRateMs` | Integer | No | Cluster topology refresh rate in milliseconds during a writer failover process. During the writer failover process, cluster topology may be refreshed at a faster pace than normal to speed up discovery of the newly promoted writer.<br/><br/>**Default value:** `5000` |
 |`failoverWriterReconnectIntervalMs` | Integer | No | Interval of time in milliseconds to wait between attempts to reconnect to a failed writer during a writer failover process.<br/><br/>**Default value:** `5000` |
 |`failoverReaderConnectTimeoutMs` | Integer | No | Maximum allowed time in milliseconds to attempt to connect to a reader instance during a reader failover process. <br/><br/>**Default value:** `5000`
-|`acceptAwsProtocolOnly` | Boolean | If using simultaneously with another postgresql driver that supports the same protocols, it is required to set this property to true either through the connection properties or at the driver level for every connection by calling Driver.setAcceptAwsProtocolOnly(true). <br/>Otherwise: No | Set to true to only accept connections for URLs with the jdbc:postgresql:aws:// protocol. This setting should be set to true when running an application that uses this driver simultaneously with another postgresql driver that supports the same protocols (eg the PostgreSQL JDBC Driver), to ensure the driver protocols do not clash.<br/><br/>**Default value:** `false`
```

Comment:
I'd recommend to add a separate section "Driver Protocol" (after "Driver Name" section) and provide info about supported/expected protocol.